Source/ChangeTracker/CBLSocketChangeTracker.m

```diff
@@ -350,4 +358,9 @@ - (void) stream: (NSStream*)stream handleEvent: (NSStreamEvent)eventCode {
 }
 
 
+- (void) handleTimeout {
+    [self failedWithErrorDomain: NSURLErrorDomain code: NSURLErrorTimedOut message: @"Timeout"];
```

Comment:
I suggest moving this up to the superclass since it's a reasonable default behavior and isn't specific to the CFStream implementation.